apps/api/src/app/workflows-v2/shared/schemas/delay-control.schema.ts

```diff
@@ -26,7 +26,7 @@ export const delayUiSchema: UiSchema = {
   properties: {
     amount: {
       component: UiComponentEnum.DELAY_AMOUNT,
-      placeholder: '30',
```

Comment:
we can't use a specific value here otherwise user will see both value and error - we don't want to support automatic default values on backend when step is inserted.

![image](https://github.com/user-attachments/assets/e4921ec9-c1bf-4a8b-be4b-c9785f9b115e)